### ISSUE TITLE
with many tries, still failed to install it?

### ISSUE THREAD

**Issue by ghost** [wdelva/RSimpactHelp#5]
> install_github("wdelva/RSimpactHelp", dependencies = TRUE)
Error in curl::curl_fetch_memory(url, handle = h) : 
  Timeout was reached: Connection timed out after 10002 milliseconds
>

**Comment by ghost** [comment]
> install_github("wdelva/RSimpactHelp", dependencies = TRUE)
Downloading GitHub repo wdelva/RSimpactHelp@master
Error in utils::download.file(url, path, method = download_method(), quiet = quiet,  : 
  download from 'https://api.github.com/repos/wdelva/RSimpactHelp/tarball/master' failed
>

**Comment by wdelva** [comment]
Hi,

Thanks for this message.
Could you share more information about what exactly the error is?
Best regards,

Wim


On 08 May 2019, at 17:15, lixingguang <notifications@github.com<mailto:notifications@github.com>> wrote:


install_github("wdelva/RSimpactHelp", dependencies = TRUE)
Downloading GitHub repo wdelva/RSimpactHelp@master
Error in utils::download.file(url, path, method = download_method(), quiet = quiet, :
download from 'https://api.github.com/repos/wdelva/RSimpactHelp/tarball/master' failed

**Comment by wdelva** [comment]
Did you follow the instructions at

https://github.com/j0r1/RSimpactCyan/blob/master/INSTALLATION.md


Best regards,

Wim



On 08 May 2019, at 5:15 PM, lixingguang <notifications@github.com<mailto:notifications@github.com>> wrote:


install_github("wdelva/RSimpactHelp", dependencies = TRUE)
Downloading GitHub repo wdelva/RSimpactHelp@master
Error in utils::download.file(url, path, method = download_method(), quiet = quiet, :
download from 'https://api.github.com/repos/wdelva/RSimpactHelp/tarball/master' failed

**Comment by ghost** [comment]
> source("https://raw.githubusercontent.com/j0r1/RSimpactCyanBootstrap/master/initsimpact.R")
Error in file(filename, "r", encoding = encoding) : 
  cannot open the connection to 'https://raw.githubusercontent.com/j0r1/RSimpactCyanBootstrap/master/initsimpact.R'
In addition: Warning message:
In file(filename, "r", encoding = encoding) :
  URL 'https://raw.githubusercontent.com/j0r1/RSimpactCyanBootstrap/master/initsimpact.R': status was 'Couldn't connect to server'

**Comment by wdelva** [comment]
Hi,

I  am beginning to think there is something generic wrong with your machine being able to access remote files on github.

Wim



On 09 May 2019, at 3:10 AM, lixingguang <notifications@github.com<mailto:notifications@github.com>> wrote:


source("https://raw.githubusercontent.com/j0r1/RSimpactCyanBootstrap/master/initsimpact.R")
Error in file(filename, "r", encoding = encoding) :
cannot open the connection to 'https://raw.githubusercontent.com/j0r1/RSimpactCyanBootstrap/master/initsimpact.R'
In addition: Warning message:
In file(filename, "r", encoding = encoding) :
URL 'https://raw.githubusercontent.com/j0r1/RSimpactCyanBootstrap/master/initsimpact.R': status was 'Couldn't connect to server'

**Comment by ghost** [comment]
Yes, how to deal with this, I know your analysis is very novel and cool, I want to use your methods for my dataset, but there are many things should to install before begging.

**Comment by wdelva** [comment]
all I can suggest is to google this problem. Are you using RStudio Server by any chance? Or a local machine?


On 09 May 2019, at 3:22 AM, lixingguang <notifications@github.com<mailto:notifications@github.com>> wrote:


Yes, how to deal with this, I know your analysis is very novel and cool, I want to use your methods for my dataset, but there are many things should to install before begging.

**Comment by ghost** [comment]
I have no RStudio Server, I just have tap.
